src/main/java/com/example/demo/Greet.java

```diff
@@ -3,10 +3,14 @@
 public class Greet {
 
     public String greet(String name) {
-        if (name != null){
-            return "Witaj, " + name;
-        } else{
+        if (name != null) {
+            if (name.equals(name.toUpperCase())) {
```

Comment:
czy można to rozwiązanie wydzielić do osobnej metody prywatnej? Czy poprawi to czytelność kodu?